### ISSUE TITLE
DPL-827 Save data against LRC Bank Stock tubes

### ISSUE THREAD

**Issue by KatyTaylor** [sanger/limber#1331]
**User story**
As a user of the scRNA Core Banking pipeline, I would like some key data to be saved against the LRC Bank Stock tubes, so that I can more easily perform the data transfer to the BioResource team in https://github.com/sanger/limber/issues/1439

**Who are the primary contacts for this story**
Lesley, Laura, Katy

**Who is the nominated tester for UAT**
Lesley, Laura (or, might be more of a technical story that we test within the team - it will then be tested when we retrieve the data in DPL-812)

**Acceptance criteria**
When the LRC PBMC Bank plate --> LRC Bank Stock tubes transfer takes place:
- [x] Cell count is stored against the LRC Bank Stock tubes as the sum of the cell counts from the source wells.
- [x] Viability is stored against the LRC Bank Stock tubes as the average (mean) of the viability from the source wells.
- [x] Volume is stored against the LRC Bank Stock tubes as the number of source wells multiplied by the volume (TBC) transferred by the robot each time 

**Dependencies**
This story is blocked by the following dependencies:
- https://github.com/sanger/limber/issues/1314

**Additional context**
Cell count and viability data is stored in the QC Results table.
Volume data is stored against the receptacle (well).

**Comment by andrewsparkes** [comment]
Questions:
1. It talks about LRC Bank Stock tubes, are these the Seq and Spare tubes?
2. How do we get the volume transferred? Is this hardcoded in the Tube purpose config? Or entered on the labware creator screen by the user?

**Comment by andrewsparkes** [comment]
Have put this story on hold whilst we develop the reporting story https://github.com/sanger/sequencescape/issues/3850
Possibly don't need to store the cell count and viability if can get the up to date QC data for that PBMC well at the point of creating a report. And volume is hardcoded to 135ul (Lesley) so no real need to store that on the tube either.
The complexity originally envisioned where we'd have to calculate values has gone now the samples are pooled BEFORE the Cellaca QC is done (pooling on Blood Bank to PBMC Bank step, then Cellaca on PBMC Bank), so no need for calculating values from multiple wells now.

**Comment by andrewsparkes** [comment]
Superceded by story https://github.com/sanger/limber/issues/1439 which fetches the information at the point of creating the report.
